common/src/unifycr_rpc_util.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2018, Lawrence Livermore National Security, LLC.
+ * Produced at the Lawrence Livermore National Laboratory.
+ *
+ * Copyright 2018, UT-Battelle, LLC.
+ *
+ * LLNL-CODE-741539
+ * All rights reserved.
+ *
+ * This is the license for UnifyCR.
+ * For details, see https://github.com/LLNL/UnifyCR.
+ * Please read https://github.com/LLNL/UnifyCR/LICENSE for full license text.
+ */
+
+#ifndef UNIFYCR_UTIL_H
+#define UNIFYCR_UTIL_H
+
+/* publish the address of the server */
+void rpc_publish_server_addr(const char* addr);
+
+/* lookup address of server */
```

Comment:
`/* lookup address of server, returns NULL if server address is not found,
 * otherwise returns server address in newly allocated string that caller must free */`

Reply:
@adammoody thanks for the review. i've updated the code.